packages/frontend/src/components/routes.jsx

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+export const HomeRoute: String = '/';
```

Comment:
If this file is not using `react`, its file extension should be `.js` for consistency.

Reply:
Okay, thanks